### DISCUSSION TITLE
Any chance at having a Docker release?

### DISCUSSION THREAD

**Discussion by spcano01** [phase1geo/Minder#609] [Ideas]
I know I can install this via WSL2, DebianVM, etc...but docker would open this project to any OS, including other Linux distros/NAS (unRAID, TrueNAS, Synology)

Installing this now via VM, but hoping for Docker option!  Thank you so much.

**Comment by phase1geo** [comment]
I don't know much about Docker. What is missing that is needed to build/run there?

**Reply by spcano01** [reply]
Docker uses containers that include all/only those dependencies to run the program and are compatible with all OS, Linux,windows, Mac, Chromebooks, etc.  

Podman, chroot, etc are similar?  Not requesting you "port" it, but hoping someone does - will make those who don't run native Linux easily be able to spin this up with much less overhead than linux VM.  

Crazy there are not more options in this space...mindmeister cannot be the be all here, happy to buy something perpetual, but they're now charging for simple things like attaching pictures since they have to host it.  I have a large server, just need a killer app! 

Thank you for building this!
